### ISSUE TITLE
Add support for contextual data input

### ISSUE THREAD

**Issue by techscientist** [maciejkula/spotlight#32]
Hi @maciejkula !

Great library! This is some seriously awesome work! 👍 

I was wondering how to add additional features that are not stationary, such as context data that applies to the whole sequence.

Can you please tell me how to do this with the Spotlight library, or is this not supported yet?

Thanks!

**Comment by nonamestreet** [comment]
I think maciejkula is actively working on it.

Maybe check out the hybrid_models branch? I'm on it and the code works.

**Comment by maciejkula** [comment]
Yes, this is the next step for Spotlight. The broad strokes of the API are ready for the factorization models in the [hybrid models](https://github.com/maciejkula/spotlight/tree/hybrid_models) branch, but no implementation has yet been written for sequence models.

It is, however, the next step!

**Comment by techscientist** [comment]
@maciejkula @nonamestreet Oh ok, nice! Thanks, and keep up the great work!

**Comment by Darenar** [comment]
Hello, @maciejkula!

I was wondering whether you have already added the possibility to use side-features in sequence models?

Thanks in advance.

**Comment by zippeurfou** [comment]
I would be interested in this as well. Treating cold start would be awesome :)
